package.json

```diff
@@ -2,12 +2,20 @@
   "private": true,
   "scripts": {
     "lint:js": "eslint .",
-    "lint:ts": "tsc"
+    "lint:ts": "tsc",
+    "test": "UNIVERSE_I18N_LOCALES=en-US meteor test-packages --port 8080 --driver-package meteortesting:mocha ./"
   },
   "devDependencies": {
+    "@types/chai": "^4.3.0",
```

Comment:
Pin all of the versions.

Reply:
Not sure what happened here but I don't think the versions are pinned?